safe/load.go

```diff
@@ -98,7 +98,7 @@ func Load(password []byte, backend backends.Client, encryptionClient crypto.Clie
 	if !reflect.DeepEqual(*safeDTO.Config, s.Config.Encryption) {
 		fmt.Println("Upgrading safe")
 		s.crypto = upgradeCrypto
-		if err := s.save(); err != nil {
+		if err := s.save(); err != nil { // nolint: vetshadow
```

Comment:
Is this shadowing due to the `err` variable in the main scope of `Load()`?

Reply:
Looks like it does. We can ignore it in this case.